packages/webpack/src/config/base.js

```diff
@@ -73,7 +73,7 @@ export default class WebpackBaseConfig {
     return [
       /\.vue\.js/i, // include SFCs in node_modules
       /consola\/src/,
-      /@nuxt[/\\]ufo/, // exports modern syntax for browser field
+      /ufo/, // exports modern syntax for browser field
```

Comment:
Should I need to change `/^ufo$/`?

Reply:
ufo should be fine. All dependant packages except tailwind seem migrated already

Reply:
I see. Thank you!